src/components/models/AppType.js

```diff
@@ -1,5 +1,4 @@
 const appType = {
-    all: "All",
```

Comment:
I removed the `All` from filters since Nirav mentioned that `Filter` `All` is confusing. With `All` option removed, users can filter by `appType` but if they want to see everything, they can simply hit `x` button next to the filter to clear it which will automatically show everything.